frontends/web/src/locales/de/app.json

```diff
@@ -598,6 +601,28 @@
       "title": "Geräteinformationen"
     },
     "expert": {
+      "bip85": {
+        "description": "Erstelle Unterschlüssel von deiner BitBox Wallet",
+        "disclaimer": "Ich verstehe, wie Unterschlüssel funktionieren und die Sicherheitsaspekte",
+        "how": {
+          "description": "Die Unterschlüssel werden als Mnemonik auf deiner BitBox angezeigt. Du kannst so viele Unterschlüssel erstellen, wie du willst. Um einen Unterschlüssel anzuzeigen, wirst du nach der Kindernummer (z.B. 1., 2., 3. usw.) und der gewünschten Länge der Mnemonik (12, 18 oder 24 Recovery-Wörter) gefragt. Denke daran, dass du die <strong>Kindernummer und die Anzahl der Wörter</strong> behalten musst, um dieselbe Mnemonik später anzeigen zu können.",
+          "title": "Wie funktioniert es?"
+        },
+        "recover": {
+          "description": "Du kannst die Mnemonik für die Unterschlüssel immer wieder anzeigen lassen. Wie erwähnt, musst du die <strong>Kindernummer und die Anzahl der Wörter</strong> merken, um dieselbe Mnemonik anzuzeigen. Importiere niemals deine Haupt-BitBox Recovery-Wörter in eine andere Wallet, verwende beim Importieren in eine andere Wallet nur Unterschlüssel.",
+          "title": "Wie kann man Wallets mit einem BitBox Unterschlüssel wiederherstellen?"
+        },
+        "security": {
+          "description": "Verwende niemals einen Unterschlüssel mehrmals. Benutze zum Beispiel nicht einen Unterschlüssel für eine Hot Wallet und dann denselben für eine Hardware-Wallet. Der Schlüssel gilt nicht als sicher, da er für eine Hot Wallet verwendet wurde.",
+          "title": "Sicherheitsaspekte"
+        },
+        "title": "Zeige BIP-85 Unterschlüssel",
+        "what": {
+          "description": "Unterschlüssel werden von deinem Haupt-Seed abgeleitet und können verwendet werden, um neue Wallets zu erstellen, ohne dass weitere Seeds gesichert werden müssen. Du kannst zum Beispiel einen Unterschlüssel erstellen und ihn in eine Hot Wallet importieren, anstatt dass die Hot Wallet einen völlig neuen Seed erstellt.",
```

Comment:
"Child Keys" leiten sich vom Haupt-Seed deiner BitBox ab und ermöglichen die Erstellung neuer Wallets, ohne dass zusätzliche Wiederherstellungswörter gesichert werden müssen. Beispielsweise kannst du einen "Child Key" generieren und in eine Hot Wallet importieren, anstatt für diese Hot Wallet vollständig neue Wiederherstellungswörter (inkl. neuem Backup) zu generieren.